core/src/main/scala/com/avast/grpc/jsonbridge/jsonbridge.scala

```diff
@@ -1,62 +0,0 @@
-package com.avast.grpc
```

Comment:
Please comment.

Reply:
The API will be completely different because the `GrpcJsonBridge` is not generic over a `BindableService` now.

Previously, the `GrpcJsonBridge` were tight to exactly one service, now it's tight to set of services.

Reply:
`DeriveBridge` class is not required because the API is different now (and we don't have to call any macro), and `ListenableFuture2ScalaFuture`'s code was moved to a new private method (because it was just one usage).